crates/steel-core/src/steel_vm/primitives.rs

```diff
@@ -1431,7 +1431,7 @@ impl crate::rvals::Custom for MutableVector {
 }
 
 struct MutableHashTable {
-    table: HashMap<SteelVal, SteelVal>,
+    table: ImmutableHashMap<SteelVal, SteelVal, FxBuildHasher>,
```

Comment:
I think this is an erroneous change - this shouldn't be changed to an immutable hash map

Reply:
Should it be FxHashMap or plain HashMap

Reply:
FxHashMap should be fine here I think